pkg/virtctl/vnc/vnc.go

```diff
@@ -39,8 +39,6 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/templates"
 )
 
```

Comment:
Thanks for your contribution.

few nits:
- would you state (in the issue, this pr) the motivation for specifying an address? I can think of a few (allowing ipv6? providing a proxy for remote vnc clients?)
- would you be kind to put the jist of the motivation in the commit message, too? It helps me when I `git log` more than just a reference to the issue you are solving.

Reply:
I can give you our specific circumstance, i figure it doesn't hurt to give users the choice anyways.

For my undergraduate project, we are building a cyber range that's scalable and open source. One requirement we had is using Apache Guacamole, a web based vnc client. We are using Harvester from openSUSE for scalability. For security reasons, we don't open any ports at all inside the VM, so the `virtctl` vnc proxy was perfect. The issue is that Guacamole is running on another machine, so locking the proxy down to localhost prevents us from achieving our goal (right now I'm using it successfully with `0.0.0.0`).

Our use case is relatively niche, but it never hurts to add another layer of customization. Another use case in mind is specifying which network adapter to limit incoming vnc traffic to by it's IP address

Let me know if you have any further questions 👍

Reply:
I added an empty commit to this PR with the message that contains the motivation. Let me know if there are any other changes needed

Reminder: This code still needs tests written to test `--address` in `vnc-test.go`. I'd rather someone who has done it before implement this so I don't mess up a GitHub project that is this important

Reply:
Thanks. I see that one of my guess (about having a non-local proxy) was correct.
Seeing an empty commit feels very awkward to me. Would you be able to squash it into the first one, so that the code change and the motivation for it come together? After you do this you may was to `git push yourremoterepo main -f` to update existing code.

Reply:
regarding tests, it is typically recommended that code and test come together in the same PR, so we never have untested code.

Would you be able to try to modify `test_id:5274` so that after it tested the `127.0.0.1` proxy, it tests your new code?